.env.example

```diff
@@ -20,59 +21,24 @@ BUGSNAG_KEY_EVENTS=123
 BUGSNAG_KEY_STATUTORY=123
 BUGSNAG_KEY_DISCOUNTS=123
 
-# Subdomains to access services
-SUBDOMAIN_FRONTEND=my.
-SUBDOMAIN_TRAEFIK=traefik.
-SUBDOMAIN_PORTAINER=portainer.
-SUBDOMAIN_PORTAL=portal.
-SUBDOMAIN_PGADMIN=pgadmin.
-SUBDOMAIN_STATUSPAGE=status.
-SUBDOMAIN_KIBANA=kibana.
-SUBDOMAIN_GRAFANA=grafana.
-SUBDOMAIN_PROMETHEUS=prom.
-SUBDOMAIN_UPMONITOR=ciao.
-SUBDOMAIN_BITWARDEN=pass.
-SUBDOMAIN_SHORTENER=a.
-SUBDOMAIN_WIKI=wiki.
-SUBDOMAIN_SURVEY=survey.
-SUBDOMAIN_UPMONITOR=ciao.
-SUBDOMAIN_STATPING=ping.
-
-# Paths in the filesystem
-PATH_OMS_GLOBAL=oms-global/docker/
-PATH_OMS_CORE_ELIXIR=oms-core-elixir/docker/
-PATH_OMS_EVENTS=oms-events/docker/
-PATH_OMS_STATUTORY=oms-statutory/docker
-PATH_OMS_FRONTEND=oms-frontend/docker/
-PATH_OMS_DISCOUNTS=oms-discounts/docker/
-PATH_OMS_MAILER=oms-mailer/docker/
-PATH_OMS_STATUS=oms-status/docker/
-PATH_OMS_SECURITY=oms-security/docker/
-PATH_OMS_SHORTENER=oms-shortener/docker/
-PATH_OMS_MONITOR=oms-monitor/docker/
-PATH_OMS_ELASTIC=oms-elastic/docker/
-PATH_OMS_WIKI=oms-wiki/docker/
-PATH_OMS_SURVEY=oms-survey/docker/
 
 CORE_LOGIN=admin
 CORE_PASSWORD=5ecr3t
 
-SMTP_HOST=smtp.gmail.com
+SMTP_HOST=smtp.gmail.com #!!!!
```

Comment:
why this?

Reply:
I don't remember and i also needed to remove it because it could break the variable